pdns/pdns_recursor.cc

```diff
@@ -1939,6 +1939,15 @@ static void startDoResolve(void *p)
         }
       }
     }
+    else if (t_pdl) {
```

Comment:
Maybe it would be better to rearrange a bit so this block ends up close to the `preresolve()` call?

Reply:
I tried, but it was too messy in the end.